packages/vhd-lib/src/merge.js

```diff
@@ -82,12 +82,11 @@ export default concurrency(2)(async function merge(
       // necessary to update values and to recreate the footer after block
       // creation
       await parentVhd.writeFooter()
-
-      return mergedDataSize
     } finally {
       await childHandler.closeFile(childFd)
     }
   } finally {
     await parentHandler.closeFile(parentFd)
   }
+  return mergedDataSize
```

Comment:
Why this change?

Reply:
probably useless, might have come from another change that got split, something with a short circuit, maybe zero block detections